app/models/champ.rb

```diff
@@ -55,6 +56,11 @@ def self.pays
 
   private
 
+  def format_to_iso_date
+    date = Date.parse(self.value).strftime("%F") rescue nil
```

Comment:
Un `if` ou un block `begin / rescue` (je ne pense pas que le `try` puisse s'appliquer dans ce cas précis).

J'aime pas car ça rend, pour moi, la lecture difficile (trop d'actions sur une meme ligne), et comme c'est en fin de ligne, on peut le rater en lecture rapide.

En plus ça a pas l'air trop recommandé : http://www.nickdesteffen.com/blog/why-rescue-nil-is-bad